src/wizard/python/profiling-onboarding/python/0.alert.md

```diff
@@ -0,0 +1,10 @@
+---
+name: Python
+doc_link: https://discord.gg/zrMjKA4Vnz
+support_level: beta
+type: language
+---
+
+<div class='alert warning'>
+Profiling using Python is currently in Beta, and there may have some bugs. We recognize the irony. If you have any questions or feedback, join our <a href="https://discord.gg/zrMjKA4Vnz">discord channel</a>.
```

Comment:
Do we want to make a distinction between alpha and beta for node/python?

Reply:
It specifies it in the alert body if its Beta or Alpha... where else would you like to define it?

Reply:
This might be me nitpicking but I believe we've been calling the upcoming release for node/python the alpha release and the message here says it's in beta.